packages/typespec-ts/test/modularIntegration/generated/azure/resource-manager/models/common-types/managed-identity/src/index.d.ts

```diff
@@ -38,7 +38,7 @@ export declare enum KnownManagedServiceIdentityType {
     None = "None",
     SystemAssigned = "SystemAssigned",
     UserAssigned = "UserAssigned",
-    SystemAndUserAssignedV3 = "SystemAssigned,UserAssigned"
+    "SystemAssigned,UserAssigned" = "SystemAssigned,UserAssigned"
```

Comment:
Is this align with hlc?

Reply:
no, it will be aligned once normalization.

![image](https://github.com/user-attachments/assets/f2406bbd-8b36-4d83-8b8b-8becd7331ebe)